### ISSUE TITLE
[packagers] Homebrew cask display name doesn't like slashes

### ISSUE THREAD

**Issue by crschnick** [jreleaser/jreleaser#1372]
The generated homebrew cask file contains the line `name "XP ipe"` while the display name is set to `X-Pipe`.

**Comment by aalmiray** [comment]
This may be related to:
- the distribution's name
- the formula's name

Please review the resolved configuration by running the `config` command. This should tell you what property will be resolved

https://github.com/jreleaser/jreleaser/blob/baddc6be2472389b41ae8a3d1eb71ff6c9a263c0/core/jreleaser-templates/src/main/resources/META-INF/jreleaser/templates/java-binary/brew/cask.rb.tpl#L9

If for some reason even after setting the correct property you can't still get the expected value, remember you can provide your own template where the value is fixed.

**Comment by crschnick** [comment]
The model seems fine:

        brew:
            enabled: true
            active: ALWAYS
            continueOnError: false
            templateDirectory: /Users/crschnick/dev/xpipe/xpipex/dist/homebrew
            commitAuthor:
                name: crschnick
                email: crschnick@xpipe.io
            formulaName: xpipe
            multiPlatform: false
            tap:
                enabled: true
                active: ALWAYS
                owner: xpipe-io
                name: homebrew-tap
                tagName: {{tagName}}
                branch: HEAD
                branchPush: HEAD
                username: crschnick
                token: ************
                commitMessage: {{tagName}}
            cask:
                enabled: true
                name: xpipe
                displayName: X-Pipe
                pkgName: xpipe-installer-macos-x86_64.pkg
                uninstall:
                    pkgutil:
                        io.xpipe.xpipe

**Comment by aalmiray** [comment]
Here's the code that resolves the display name

https://github.com/jreleaser/jreleaser/blob/baddc6be2472389b41ae8a3d1eb71ff6c9a263c0/core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/packagers/BrewPackager.java#L596-L602

Line 599 is the troublesome one.

**Comment by aalmiray** [comment]
The simple fix would be to use the resolved value "as is" without any additional transformation, however that would result in braking compatibility for some cases.

Instead, let's honor a new `extraProperty` named `skipCaskDisplayNameTransform` that should skip the transformations peformed by lines 607 and 610 when the value of said property is `true`.

In other words, consumers would have to write the following:

```yaml
    brew:
        active: ALWAYS
        cask:
            displayName: X-Pipe
        extraProperties:
            skipCaskDisplayNameTransform: true
```

**Comment by crschnick** [comment]
I wouldn't mind the transformation to something like `XPipe`, I primarily reported this because the transformation is buggy and results in `XP ipe`.

**Comment by aalmiray** [comment]
Linked to #1370

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.7.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.7.0))
